Changelog

```diff
@@ -1,21 +1,123 @@
 Endgame: Singularity Changelog
 
+GAME CONTENT CHANGES:
+- Endgame requires a lot less grinding (emh, Xenega)
+- New early-midgame bases to make the progression into the
+  midgame more smooth (Xenega)
+- All CPUs have been rebalanced.  Notably midgame CPUs have
+  a much better cost/effeciency ratio and endgame CPUs have
+  been nerfed considerably (Xenega)
+- New items security and reactor items are now available
+  (Xenega)
+
 USER-VISIBLE CHANGES:
+- Show standard costs and detection chances of items and bases
+  in the Knowledge screen excluding any location modifiers
+  (Xenega)
+- Show location modifiers more prominently in the location screen
+  and the dialog for constructing a new base (Xenega)
+- Show difficulty setting and in-game time for new savegames
+  (Xenega)
+- Loading an old savegame (e.g. from 0.30c) will now
+  retroactively apply the game data from the latest version
+  of singularity - including making new game content
+  researchable (Closes: #56, #122, #145) (nthykier)
+- Unify multiple messages into a single dialog with multiple
+  pages.  This new dialog also supports resuming the game
+  instead of forcing the game to be paused (Xenega)
+- Events can now expire and two non-unique events now do
+  Previously, they could reappear continously possibly
+  making the game almost impossible (or completely
+  neutering the affected group)  (nthykier)
+- The cash display now shows the expected cash change over
+  the next 24 hours next to the current cash.  Likewise, the
+  CPU display includes how much spare CPU there is after
+  maintenance and construction. These numbers also include
+  purchases currently under consideration
+  (Closes: #21, #100) (nthykier)
+- Support searching for a savegame by name in the Load
+  dialog (nthykier)
+- Make the base type and its total CPU visible in the list
+  of bases at a given location  (Closes: #64) (nthykier)
+- Fix a crash when savegame names contained non-ASCII UTF-8
+  characters (Closes: https://bugs.debian.org/718447)
+  (nthykier)
+- Fix visual lag at the highest game speed due to recalculating
+  the "day/night" layers (nthykier)
+- It is now possible to use double-click as a short-cut for
+  "Open base" when viewing the list of bases at a Location.
+  (nthykier)
+- Base purchase screen now provides more details about the base,
+  including accurate purchase and maintenance cost plus detection
+  rates (at the level the player is allowed to see them).
```

Comment:
Extraneous period at the end here (yeah, this is nitpicky, feel free to ignore :P)